File: firstdegreeequation.php
<?php 
	function calculate($a,$b){
		if ($a == 0) {
			if ($b == 0) {
				echo "Phương trình có vô số nghiệm";
			}else{
				echo "Phương trình vô nghiệm";
			}
		}else{
			echo "Phương trình có nghiệm: " . -$b/$a;
		}
	}

	calculate(1,1);
	calculate(0,6);
	calculate(0,0);
	calculate(3,3);

 ?>




"""@author: github.com/isacarcanjo"""

File: firstdegreeequation.java
import java.util.Scanner;
import java.io.File;
public class file
{
   public static void main(String[] args)
   {
       Scanner myScanner = null;
       try
       {
           myScanner = new Scanner(new File("test.txt"));
           while (myScanner.hasNextLine())
           {
               System.out.println(myScanner.nextLine());
           }
       }
       catch (Exception ex)
       {
           ex.printStackTrace();
       }
       finally
       {
           if (myScanner != null)
           {
               myScanner.close();
           }
       }
   }
}




"""@author: github.com/isacarcanjo"""



